suites/squid/rados/tier-2_rados_test-brownfield.yaml

```diff
@@ -0,0 +1,221 @@
+# Suite contains tests to be run after upgrade [brownfield deployment]
+# Use cluster-conf file: conf/squid/rados/7-node-cluster.yaml
+
+tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: cluster deployment
+      desc: Execute the cluster deployment workflow.
+      module: test_cephadm.py
+      polarion-id:
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              base_cmd_args:
+                verbose: true
+              args:
+                rhcs-version: 6.1
```

Comment:
Let's change this to 7.1, so we do not have to amend later

Reply:
Made the changes as per the comments.